docs/vsb_api_ref.md

```diff
@@ -2,29 +2,31 @@
 
-| Property             | Type                       | Description                                        |
-|----------------------|--------------------------------|-------------------------------------------------------|
-| VolumeSnapshotContent      | corev1.ObjectReference                   | VolumeSnapshotContent is the name of the VolumeSnapshotContent that will be moved to a remote storage location.          |
-| ProtectedNamespace      | string                 | ProtectedNamespace is the namespace in which the Velero deployment is present, and where VolumeSnapshotBackup resources will be created.   |
+| Property              | Type                       | Description                                        |
+|-----------------------|--------------------------------|-------------------------------------------------------|
+| VolumeSnapshotContent | corev1.ObjectReference                   | VolumeSnapshotContent is the name of the VolumeSnapshotContent that will be moved to a remote storage location.          |
+| ProtectedNamespace    | string                 | ProtectedNamespace is the namespace in which the Velero deployment is present, and where VolumeSnapshotBackup resources will be created.   |
+| ResticSecretRef       | corev1.LocalObjectReference                 | Restic Secret reference for given BSL  |
 
 
 ### VolumeSnapshotBackupStatus
 
-| Property             | Type                        | Description                                 |
-|----------------------|-------------------------------------------------|------------------------------------------------------|
-| Completed     | bool                                                    | Completed is whether or not VolumeSnapshotBackup has completed reconciling.    |
-| SourcePVCData      | PVCData                                                | SourcePVCData is a reference to the source PVC.            |
-| ResticRepository      | string                                             | ResticRepository is the location in which the snapshot will be stored.        |
-| Phase      | VolumeSnapshotBackupPhase                                       | Phase is the VolumeSnapshotBackup phase status.           |
-
+| Property             | Type                      | Description                                                                 |
+|----------------------|---------------------------|-----------------------------------------------------------------------------|
+| Completed     | bool                      | Completed is whether or not VolumeSnapshotBackup has completed reconciling. |
+| SourcePVCData      | PVCData                   | SourcePVCData is a reference to the source PVC.                             |
+| ResticRepository      | string                    | ResticRepository is the location in which the snapshot will be stored.      |
+| Phase      | VolumeSnapshotBackupPhase | Phase is the VolumeSnapshotBackup phase status.                             |
```

Comment:
`Phase` is what we update with VolSync CR state as they are running, right?

Reply:
Not exactly, Volsync CR states would be a subset of datamover phases, right ?